WORKSPACE

```diff
@@ -168,9 +168,19 @@ http_archive(
 
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
+git_repository(
+    name = "io_bazel_stardoc",
+    remote = "https://github.com/bazelbuild/stardoc.git",
+    tag = "0.4.0",
+)
+
+load("@io_bazel_stardoc//:setup.bzl", "stardoc_repositories")
+
+stardoc_repositories()
```

Comment:
Not including this apparently is an issue.  We might not have the problem if we were using the rules_kotlin from head (and using http_repository).. but this is a minor thing to add IMO.